tests/spread/plugins/go/snaps/go-use-build-packages/snap/snapcraft.yaml

```diff
@@ -0,0 +1,20 @@
+name: go-hello
+version: 0.1
+summary: A simple go project
+description: |
+  This is a basic go snap. It just prints a hello world and checksthat it is
+  possible to build a snap using local go sources with go from build-packages.
+
+grade: devel
+confinement: strict
+
+apps:
+  go-hello:
+    command: go-hello
+
+parts:
+  go-hello:
+    plugin: go
+    source: go-hello
+    build-packages:
+      -golang-go
```

Comment:
No space after the hyphen is bothering my OCD. It looks like a negative.

Reply:
yeah, it is not even an array :-)